web/client/components/map/openlayers/Layer.jsx

```diff
@@ -123,9 +127,35 @@ class OpenlayersLayer extends React.Component {
         if (type) {
             const layerOptions = this.generateOpts(options, position, CoordinatesUtils.normalizeSRS(this.props.srs), securityToken);
             this.layer = Layers.createLayer(type, layerOptions, this.props.map, this.props.mapId);
+
             if (this.layer && !this.layer.detached) {
+                const parentMap = this.layer && this.layer.getProperties().map;
```

Comment:
every layer has a this.props.map property that stores the openlayers map, this is not needed.

Reply:
ok, solved